<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNB</class>
 <widget class="QDialog" name="DialogNB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>683</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_NB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Naive Bayes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="nb_return">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>630</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="nb_filecontext">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>601</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nb_fileno_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>135</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Test File No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_stem_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>135</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stem:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nb_getfile">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>135</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_dataset_label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LargeDataSet:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nb_stem">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="nb_dataset">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nb_type_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nb_type_spam">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Spam</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="nb_type_ham">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ham</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="nb_fileno">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>131</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>250</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nb_dataset_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DataSet Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_dataset_show">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Null</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_trainfilenum_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Train File Num:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_trainfilenum_show">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Null</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_stemmed_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stemmed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_stemmed_show">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Null</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_stopword_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ban StopWord:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nb_stopword">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nb_run">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_hamac_label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ham Accuracy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_spamac_label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Spam Accuracy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_totalac_label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total Accuracy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_hamac_show">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nb_spamac_show">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>120</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nb_totalac_show">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nb_classification_res_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>620</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Classification Result:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nb_classification_res_show">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>620</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nb_classification_res_show2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>620</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nb_fileno_range">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>nb_totalac_label</zorder>
  <zorder>nb_spamac_label</zorder>
  <zorder>nb_hamac_label</zorder>
  <zorder>nb_dataset_label</zorder>
  <zorder>nb_trainfilenum_label</zorder>
  <zorder>nb_stemmed_label</zorder>
  <zorder>nb_stopword_label</zorder>
  <zorder>nb_dataset_label_2</zorder>
  <zorder>nb_fileno_label</zorder>
  <zorder>nb_stem_label</zorder>
  <zorder>label_NB</zorder>
  <zorder>nb_return</zorder>
  <zorder>nb_filecontext</zorder>
  <zorder>nb_getfile</zorder>
  <zorder>nb_stem</zorder>
  <zorder>nb_dataset</zorder>
  <zorder>nb_type_label</zorder>
  <zorder>nb_type_spam</zorder>
  <zorder>nb_type_ham</zorder>
  <zorder>nb_fileno</zorder>
  <zorder>customPlot</zorder>
  <zorder>nb_dataset_show</zorder>
  <zorder>nb_trainfilenum_show</zorder>
  <zorder>nb_stemmed_show</zorder>
  <zorder>nb_stopword</zorder>
  <zorder>nb_run</zorder>
  <zorder>nb_hamac_show</zorder>
  <zorder>nb_spamac_show</zorder>
  <zorder>nb_totalac_show</zorder>
  <zorder>nb_classification_res_label</zorder>
  <zorder>nb_classification_res_show</zorder>
  <zorder>nb_classification_res_show2</zorder>
  <zorder>nb_fileno_range</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
